client/src/App.js

```diff
@@ -4,13 +4,15 @@ import Student from "./pages/Student";
 import LoginPage from "./pages/LoginPage";
 import NavBar from "./components/NavBar";
 import Footer from "./components/Footer";
+import { Admin } from "./pages/Admin";
```

Comment:
do we need the selection brackets here? See line 4.

(!this won't cause any actual issues!)

Reply:
Thanks for pointing it out. I think the import was done automatically that's why we've got the brackets. It won't affect, but I've already removed them, the changes will appear in my next PR..